schema-loader/src/main/java/com/scalar/db/schemaloader/SchemaLoader.java

```diff
@@ -377,6 +381,129 @@ private static void repairTables(
     }
   }
 
+  /**
+   * Alter the tables defined in the schema. Supported alteration are :
+   *
+   * <ul>
+   *   <li>add non primary-key columns
+   *   <li>create secondary indexes
+   *   <li>delete secondary indexes
+   * </ul>
+   *
+   * By comparing the differences between the provided schema and the current schema, it will know
+   * which columns should be added and which secondary indexes should be created or deleted. Only
+   * the tables with differences will be altered.
+   *
+   * @param configProperties Scalar DB config properties
+   * @param serializedSchemaJson serialized json string schema.
+   * @param indexCreationOptions specific options for index creation.
+   * @throws SchemaLoaderException thrown when altering tables failed.
+   */
+  public static void alterTables(
+      Properties configProperties,
+      String serializedSchemaJson,
+      Map<String, String> indexCreationOptions)
+      throws SchemaLoaderException {
+    Either<Path, Properties> config = new Right<>(configProperties);
+    Either<Path, String> schema = new Right<>(serializedSchemaJson);
+    alterTables(config, schema, indexCreationOptions);
+  }
+
+  /**
+   * Alter the tables defined in the schema. Supported alteration are :
```

Comment:
```suggestion
   * Alter the tables defined in the schema. Supported alterations are :
```
Or, `supported alter operations are:`?
Ditto for the following comments.

Reply:
Ok, I will replace it with `supported alter operations are:`